x-pack/plugins/observability_solution/apm/public/components/shared/charts/helper/get_chart_anomaly_timeseries.tsx

```diff
@@ -69,15 +70,15 @@ export function getChartAnomalyTimeseries({
   const scores: APMChartSpec[] = severities.map(({ severity, threshold }) => {
     const color = getSeverityColor(threshold);
 
-    const style = {
+    const style: RecursivePartial<AreaSeriesStyle> = {
       line: {
         opacity: 0,
       },
       area: {
         fill: color,
       },
       point: {
-        visible: true,
+        visible: 'never',
```

Comment:
Shouldn't it be `always`?

Reply:
ups, sorry, will fix it immediately

Reply:
done in https://github.com/elastic/kibana/pull/193694/commits/718eb0b965c5c16122995ac8c32e8554a219012b